test/exercises/2023JanTutorial/kmer.chpl

```diff
@@ -0,0 +1,48 @@
+// kmer.chpl
+/*
+   usage on puma/ocelote:
+     chpl kmer.chpl
+     ./kmer
+
+   usage on laptop with podman (or docker):
+     podman run --rm -v "$PWD":/myapp -w /myapp chapel/chapel chpl kmer.chpl
+     podman run --rm -v "$PWD":/myapp -w /myapp chapel/chapel ./kmer
+
+   For docker usage, see https://chapel-lang.org/install-docker.html
+
+   Original version of kmer counting algorithm provided by
+   Aryamaan Jain (github strikeraryu) on Chapel Discourse April 7, 2021.
+
+   kmer_large_input.txt is from
+   https://www.ncbi.nlm.nih.gov/nuccore/NC_001422.1?report=fasta
+ */
+
+use Map;
+use IO;
+
+// to have it read a different input file, run as follows:
+//      ./kmer --infile="anotherFileName"
+config const infile = "kmer_large_input.txt";
+
+// set k to something different on the commandline with
+//      ./kmer --k=7
+config const k = 4;
+
+// read in the input sequence from the file infile
+var sequence : string;
+var f = open(infile, iomode.r);
+var fReader =  f.reader();
```

Comment:
If desired, I believe you could just write this as `var fReader = open(infile. iomode.r).reader();` 

`fReader` also strikes me as a clunky name (and `f` as an unusually short one for this program).  I usually call my reader/writer channels something like `infile` and the filename `infilename`.

Reply:
Partially done.  I did the renaming, but I am keeping things split out to make them easier to initially understand.